docs/modules/network-partitioning/pages/split-brain-recovery.adoc

```diff
@@ -550,4 +551,48 @@ since they are already in the correct in-memory format of the data structure.
 If you return a deserialized value, it might need to be serialized again, which are avoidable costs.
 * Be careful with slow operations in the merge policy (like database accesses),
 since they block your partition threads. Also the `LifeCycleEvent.MERGED` or
-`LifeCycleEvent.MERGE_FAILED` may be thrown too early, if the merge operations take too long to finish.
\ No newline at end of file
+`LifeCycleEvent.MERGE_FAILED` may be thrown too early, if the merge operations take too long to finish.
+
+== Eliminating Unsuccessful Cluster Merges
+
+In some cases, Hazelcast Platform members which initially can form a cluster (join to each other) may not be
+able to recover from the split-brain even though the network failure which caused the split-brain is healed.
+The reason is that the master member of the smaller cluster fails to discover the master member of the bigger cluster, so the split-brain merge process never begins.
+
+Considering a scale-up situation; you add new members after some time since the initial member count in the cluster has become
+insufficient. The IP addresses of these new members may not exist in the configurations of
+the initial set of members, since the new addresses are not added into the the configurations of the initial members while starting them.
+So, in a cluster with a long lifecycle, some members may not have the address of every other member, resulting in split-brain recovery failures.
+
+To overcome this issue, you can use the following REST endpoint to update the member list on the cluster members at runtime.
+
+[source,shell]
+----
+hazelcast/rest/config/tcp-ip/member-list
+----
+
+You can send a `GET` request to the above endpoint to retrieve the current member list from the member and a `POST` request for updating the member list.
+
+The following is the cURL command to retrieve the member list, and its response format:
+
+[source,shell]
+----
+curl -X GET http://<member IP address>:<port>/hazelcast/rest/config/tcp-ip/member-list
+
+{"status":"success","member-list":["<member-1-address>","<member-2-address>","<member-3-address>", ...]}
+----
+
+The following is the cURL command to update the member list, and its response format:
+
+[source,shell]
+----
+curl -X POST -H "Content-Type: text/plain" --data-urlencode "<cluster-name>" --data-urlencode "<cluster-password>" --data-urlencode "<member1-address>,<member2-address>,<member3-address>" http://<member IP address>:<port>/hazelcast/rest/config/tcp-ip/member-list
+
+{"status":"success","message":"The member list of TCP-IP join config is updated at run time. ","member-list":["<member1-address>","<member2-address>","<member3-address>"]
```

Comment:
I would use a simpler form here.
```suggestion
curl --data-urlencode "<cluster-name>" --data-urlencode "" --data-urlencode "<member1-address>,<member2-address>,<member3-address>" http://<member IP address>:<port>/hazelcast/rest/config/tcp-ip/member-list

{"status":"success","message":"The member list of TCP-IP join config is updated at run time. ","member-list":["<member1-address>","<member2-address>","<member3-address>"]}
```

Then I would add a note for the first and second arguments. If security is enabled and a custom **member** authentication is used, then the first argument is the username, and the second is the password.